app/graphql/types/query.rb

```diff
@@ -23,32 +23,15 @@ def collection_field(name, type)
     record_field :profile, Types::Profile
     record_field :test_result, Types::TestResult
 
-    field :all_systems, [Types::System], null: true do
-      description 'All systems visible by the user'
-      argument :search, String, 'Search query', required: false
-      argument :per_page, Int, 'Page', required: false
-      argument :page, Int, 'Per page', required: false
-      argument :profile_id, String, 'Profile Id', required: false
-    end
-
     field :system, Types::System, null: true do
       description 'Details for a system'
       argument :id, String, required: true
     end
 
-    field :all_image_streams, [Types::System], null: true do
-      description 'All image streams visible by the user'
-      argument :search, String, 'Search query', required: false
-    end
-
     field :all_profiles, [Types::Profile], null: true do
```

Comment:
This one will be removed soon as well, right?

Reply:
Yup, I've seen it used on 1 place still though